doc/conf.py

```diff
@@ -24,11 +24,11 @@
 # -- Project information -----------------------------------------------------
 
 project = 'hnn-core'
-copyright = '2019, Mainak Jas'
-author = 'Mainak Jas'
+copyright = '2020, HNN Developers'
```

Comment:
Any reason this isn't the current year? Honestly just curious...

Reply:
honestly not sure but I guess it should be current year :)